### PR TITLE
Move gracc-apel to opensciencegrid/images (SOFTWARE-5054)

### PR DESCRIPTION
**Pull request by yongleyuan**

### CODE REVIEW

File: opensciencegrid/gracc-apel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+FROM opensciencegrid/osg-wn:3.5-el7
```
Comment (brianhlin) [inline opensciencegrid/gracc-apel/Dockerfile:1]: This is odd @djw8605 is there a reason that we inherit from the WN client instead of `software-base`?

Comment (djw8605) [reply opensciencegrid/gracc-apel/Dockerfile:1]: Probably just a bad decision at the time. Feel free to update!

Comment (brianhlin) [reply opensciencegrid/gracc-apel/Dockerfile:1]: @yongleyuan please update the base to look like the other Dockerfiles that you've moved over

Comment (yongleyuan) [reply opensciencegrid/gracc-apel/Dockerfile]: Just updated it in this commit: https://github.com/opensciencegrid/images/pull/12/commits/0147944596335f73369c628b1eabcf079c7a7219.

File: opensciencegrid/gracc-apel/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  pip3 install elasticsearch-dsl && \
+  pip3 install argo-ams-library
+RUN yum -y install http://rpm-repo.argo.grnet.gr/ARGO/devel/centos7/python-argo-ams-library-0.5.5-20210415071520.ff0c536.el7.noarch.rpm
+
+RUN yum -y install python-dirq
+
+# install ca-policy-egi-core
+RUN yum -y remove osg-ca-certs
+RUN cd /etc/yum.repos.d/ && \
+  wget http://repository.egi.eu/sw/production/cas/1/current/repo-files/EGI-trustanchors.repo && \
```
Comment (brianhlin) [inline opensciencegrid/gracc-apel/Dockerfile:16]:
```suggestion
RUN curl -so /etc/yum.repos.d/EGI-trustanchors.repo https://repository.egi.eu/sw/production/cas/1/current/repo-files/EGI-trustanchors.repo && \
```

`wget` isn't in `software-base`; also use HTTPS

Comment (brianhlin) [reply opensciencegrid/gracc-apel/Dockerfile:16]: @yongleyuan what's the status on this one? It's been nearly 3 weeks!

**Comment by brianhlin** [comment]
Could you also remove the `.github` subdirectory?

**Review by brianhlin — APPROVED** [review]
LGTM, the issue with too many CI tests should have been fixed in #30

**Pull request merged**